README.md

```diff
@@ -18,10 +18,10 @@
 ### Requirements
 
 Supported platforms:
-* Unix-like os, tested on Linux and macOS; Windows is currently not supported
-* Node.js 14, 16 or 18
-* Express.js
-* git is highly recommended 
+* Mainstream os: linux distros, osx, and windows
```

Comment:
Why not leave this as "macOS", i.e. the official name Apple uses?

Reply:
I don't know why I changed it, I revert it 36718ff399386ac0feec8b889e65044c8c3d15af